### PR TITLE
Add documentation for Tami4

### PR DESCRIPTION
**Pull request by Guy293**
## Proposed change

Add documentation for Tami4 integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/90056
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3836

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Guy293** [comment]
@klaasnicolaas Fixed it, my bad.

**Comment by Quentame** [comment]
You need to add a Brands PR.

Is this PR the same ? https://github.com/home-assistant/brands/pull/3599
If yes rename the folder to tami4, and make an alias for the custom component

**Comment by Guy293** [comment]
> You need to add a Brands PR.
> 
> Is this PR the same ? [home-assistant/brands#3599](https://github.com/home-assistant/brands/pull/3599) If yes rename the folder to tami4, and make an alias for the custom component

Yeah, It's someone's else custom integration based on my Python module.
Will there be a problem if I'll just use his uploaded brand?
Or should I copy it to another folder names tami4?

**Comment by Quentame** [comment]
> > You need to add a Brands PR.
> > Is this PR the same ? [home-assistant/brands#3599](https://github.com/home-assistant/brands/pull/3599) If yes rename the folder to tami4, and make an alias for the custom component
> 
> Yeah, It's someone's else custom integration based on my Python module. Will there be a problem if I'll just use his uploaded brand? Or should I copy it to another folder names tami4?

Nope, you totally can use his images !

As I said in my first message: If it's the same integration **rename** the folder to tami4, and make an alias (a.k.a symlink like https://github.com/home-assistant/brands/pull/1130) for the custom component

So it will be like:
- `tami4` folder -> contains brand images
- `tami4edge` folder wich is a symlink of `tami4` folder

File: source/_integrations/tami4.markdown [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Tami4 Edge / Edge+"
+description: "Tami4 Edge Integration"
+ha_release: "2022.11"
+ha_category: Other
```
Comment (Quentame) [inline source/_integrations/tami4.markdown]:
```suggestion
ha_category: Sensor
```

File: source/_integrations/tami4.markdown [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Tami4 Edge / Edge+"
+description: "Tami4 Edge Integration"
+ha_release: "2022.11"
```
Comment (Quentame) [inline source/_integrations/tami4.markdown]:
```suggestion
ha_release: 2022.12
```

`"` is required only when the end number ends by a 0

**Review by Quentame — CHANGES_REQUESTED** [review]

File: source/_integrations/tami4.markdown [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+ha_codeowners:
+  - '@Guy293'
+ha_domain: tami4
+---
+
+Support for Tami4 Edge / Edge+ by Strauss in Home Assistant.
+
+You can boil water and get filter / uv information, such as when you need to replace them.
+
+You can easily pair your device by confirming your phone number which is registered with your Edge app account.
```
Comment (Quentame) [inline source/_integrations/tami4.markdown:17]:
```suggestion
You can easily pair your device by confirming your phone number which is registered with your Edge app account.

{% include integrations/config_flow.md %}
```
It misses the "how to configure" section + I'm not sure this sentence really help to configure the service into Home Assistant.

Either:
- create a section before the HA configuration to explain how to configure your Tami4 account
- explain only the HA configuration part

File: source/_integrations/tami4.markdown [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+ha_category: Other
+ha_iot_class: "Cloud Polling"
+ha_quality_scale: silver
+ha_config_flow: true
+ha_codeowners:
+  - '@Guy293'
+ha_domain: tami4
+---
+
+Support for Tami4 Edge / Edge+ by Strauss in Home Assistant.
```
Comment (Quentame) [inline source/_integrations/tami4.markdown]:
```suggestion
Support for [Tami4 Edge / Edge+ by Strauss](https://www.tami4.co.il/tami4edge-collection) in Home Assistant.
```
You should add the link to the official website, I hope I put the right one.

File: source/_integrations/tami4.markdown (hunk 1 above)
Comment (Guy293) [reply source/_integrations/tami4.markdown]: It's not exactly a sensor, it also controls the device

Comment (Quentame) [reply source/_integrations/tami4.markdown]: This attribute can be a list, on your current PR there is only sensor platform, you can still add other platforms later

File: source/_integrations/tami4.markdown [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+Support for [Tami4 Edge / Edge+ by Strauss](https://www.tami4.co.il/tami4edge-collection) in Home Assistant.
+
+You can boil water and get filter / UV information, such as when you need to replace them.
+
+You can easily pair your device by confirming your phone number which is registered with your Edge app account.
+
+## Configuration
+
+Add a new Integration in Home Assistant and select Tami4.
+Then, enter your phone number which use used to register
```
Comment (Quentame) [inline source/_integrations/tami4.markdown]:
```suggestion
Then, enter your phone number which you used to register
```

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

File: source/_integrations/tami4.markdown [hunk 6]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Tami4 Edge / Edge+"
+description: "Tami4 Edge Integration"
+ha_release: 2022.12
```
Comment (edenhaus) [inline source/_integrations/tami4.markdown]:
```suggestion
ha_release: 2023.11
```

File: source/_integrations/tami4.markdown [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_quality_scale: silver
```
Comment (edenhaus) [inline source/_integrations/tami4.markdown]:
```suggestion
```

Integration has no score

**Review by edenhaus — CHANGES_REQUESTED** [review]

File: source/_integrations/tami4.markdown [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+You can boil water and get filter / UV information, such as when you need to replace them.
+
+You can easily pair your device by confirming your phone number which is registered with your Edge app account.
+
+## Configuration
+
+Add a new Integration in Home Assistant and select Tami4.
+Then, enter your phone number which you used to register
+to the Edge app and confirm the code with the SMS you've received.
```
Comment (frenck) [inline source/_integrations/tami4.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

**Review by frenck — APPROVED** [review]

**Pull request merged**